File: sample/src/main/java/cn/dong/viewbinder/sample/MainActivity.java
package cn.dong.viewbinder.sample;

import android.os.Bundle;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;
import cn.dong.viewbinder.ViewBinder;
import cn.dong.viewbinder.annotation.BindView;

public class MainActivity extends AppCompatActivity {
    @BindView(R.id.hello) TextView helloText;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ViewBinder.bind(this);
        helloText.setText("hello");
    }
}

File: sample_runtime/src/main/java/cn/dong/viewbinder/sample/runtime/MainActivity.kt
package cn.dong.viewbinder.sample.runtime

import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

@ContentView(R.layout.activity_main)
class MainActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        hello.text = "hello"
    }
}
